### PR TITLE
Added Courses

### PR DESCRIPTION
**Pull request by ivikashgorai**
## What does this PR do?
Add resource(s) 

### Why is this valuable (or not)?
It Covers all topic from beginner to advance 

### How do we know it's really free?
Available in youtube

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -881,6 +881,7 @@
 * [Java Tutorial for Beginners](https://www.youtube.com/watch?v=eIrMbAQSU34) - Programming with Mosh
 * [Java Tutorial For Beginners](https://youtube.com/playlist?list=PLsyeobzWxl7oZ-fxDYkOToURHhMuWD1BK) - Telusko
 * [Java Tutorial for Complete Beginners](https://www.udemy.com/course/java-tutorial/) - John Purcell (Udemy)
+* [Java Zero to FAANG](https://www.youtube.com/playlist?list=PL9gnSGHSqcnr_DxHsP7AW9ftq0AtAyYqJ) - Kunal Kushwaha
```
Comment (eshellman) [inline courses/free-courses-en.md:884]: Don't invent titles! Use the title given by the resource.

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -480,8 +481,9 @@
 
 #### Redux
 
+* [Redux Complete Course](https://youtube.com/playlist?list=PL8p2I9GklV47TDYUq8RmFzeI9CgOoVgpJ&si=p-4s6qJ31ReIbKqj) - Code Step by Step
 * [Redux toolkit crash course &#124; Chai aur React Series](https://www.youtube.com/watch?v=1i04-A7kfFI) - Hitesh Choudhary (Chai aur Code)
-* [Redux Toolkit tutorial in Hindi](https://www.youtube.com/playlist?list=PLwGdqUZWnOp2nz2T6SfWX9t6D6SYn3XlN) - Vinod Bahadur Thapa (Thapa Technical)
```
Comment (eshellman) [inline courses/free-courses-hi.md:484]: did you mean to remove this???

File: courses/free-courses-hi.md [hunk 3]
```diff
@@ -480,8 +481,9 @@
 
 #### Redux
 
+* [Redux Complete Course](https://youtube.com/playlist?list=PL8p2I9GklV47TDYUq8RmFzeI9CgOoVgpJ&si=p-4s6qJ31ReIbKqj) - Code Step by Step
```
Comment (eshellman) [inline courses/free-courses-hi.md:484]: Don't invent titles! Use the title given by the resource.

**Review by eshellman — CHANGES_REQUESTED** [review]

File: courses/free-courses-hi.md (hunk 2 above)
Comment (ivikashgorai) [reply courses/free-courses-hi.md:484]: sorry my mistake

**Comment by ivikashgorai** [comment]
Please review and merge it

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**